vcs/queries.py

```diff
@@ -105,16 +104,18 @@ def graphicsmethodlist():
 
     :Example:
 
-::
+        .. doctest:: queries_gmlist
 
-    a=vcs.init()
-    # Return graphics method list
-    gm_list=a.graphicsmethodlist()
+            >>> a=vcs.init()
+            ...
 
-:returns: A list of available grapics methods (i.e., boxfill, isofill, isoline, outfill,
-          scatter, vector, xvsy, xyvsy, yxvsx, taylordiagram ).
-:rtype: list
-"""
+            >>> vcs.graphicsmethodlist() # Return graphics method list
+            ['boxfill', 'isofill', 'isoline', 'meshfill', 'scatter', 'vector', 'xvsy', 'xyvsy', 'yxvsx', ...]
```

Comment:
@doutriaux1 
Doctests have you supply a line of code of the format:
```python
>>> object.method()
'Return value'
```
When I call vcs.graphicsmethodlist() in a python console, it gives me the specified array.
The reason that I have the ```...``` at the end is that the array is too large to fit in 120 characters, and doctest will throw an error if I dont have the exact array, unless I throw ```...``` in there, which is kind of like a wildcard "match anything here" statement.
My thought was that the examples are to give a representation of what expected output should generally look like, not the exact details.
'xvsy', 'xyvsy', 'yxvsx', as well as '1d' and other things I don't show here show up in my array, when I call graphicsmethodlist().